CONTRIBUTING.md

```diff
@@ -19,6 +15,25 @@ This repo uses short-lived, prefixed branches. For example, a new blog post woul
 * [Search](#search)
 * [Translating](#translating)
 
+## Branching Processes
+
+Pull requests should be made against the `develop` branch.
+
+This repo uses short-lived, prefixed branches. For example, a new blog post would live a branch named something like: `content/cool-truffle-news`. This helps multiple teams touch the site if necessary, while mitigating potential collisions. These are the currently used prefixes:
+
+`content`: edits applying to site content.
+`feature`: new features for the website, including new data structures (e.g. blog posts or events).
+`fix`: fixes an error in existing code.
+
+## Merging Processes
+
+1. As mentioned above, your contribution will be a branch based off of `develop`
+1. Once it's ready for review, please create a new [Pull Request](https://github.com/trufflesuite/trufflesuite.com/compare) to be merged into (aka base branch) `develop`
+1. PRs from Truffle employee require at least one review approval, PRs from external contributors are recommended (though not required) to have >1 review
```

Comment:
"Member" is probably a better noun than "Truffle employee" (more consistent-with-GH-terms), if it's easy to make sound alright

Reply:
ah also: two separate sentences are joined by semicolons:
```suggestion
1. PRs from Truffle employee require at least one review approval; PRs from external contributors are recommended (though not required) to have >1 review
```

comma alternative:
```suggestion
1. PRs from Truffle employee require at least one review approval, and PRs from external contributors are recommended (though not required) to have >1 review
```